VARIABLES.md

```diff
@@ -636,6 +660,14 @@ Default:  "{{ ssl_mutual_auth_enabled }}"
 
 ***
 
+### kafka_rest_log_dir
+
+Only use to customize directory Rest Proxy writes log files to. Default location is /var/log/confluent/kafka-rest. NOTE- kafka_rest.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that the Rest Proxy writes log files to. Default location is /var/log/confluent/kafka-rest. NOTE- kafka_rest.appender_log_path is deprecated.